ignite/pkg/availableport/availableport.go

```diff
@@ -16,8 +16,8 @@ func Find(n int) (ports []int, err error) {
 
 	for i := 0; i < n; i++ {
 		for {
-			rand.Seed(time.Now().UnixNano())
-			port := rand.Intn(max-min+1) + min
+			r := rand.New(rand.NewSource(time.Now().UnixNano()))
+			port := r.Intn(max-min+1) + min
```

Comment:
Since 1.20 Go automatically seeds the global random number generator so in this specific case this could be replaced by:

```suggestion
			port := rand.Intn(max-min+1) + min
```